components/dashboard/src/projects/NewProject.tsx

```diff
@@ -406,15 +407,9 @@ export default function NewProject() {
         </div>);
     }
 
-    const renderSelectRepoHeading = () => {
-        return <p className="text-gray-500 text-center text-base">Select a Git repository on <strong>{provider}</strong>. (<a className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>change</a>)</p>
-    }
```

Comment:
Self-nit: This refactoring removed the heading from the repo loading screen. Okay / not okay?

Reply:
It's probably fine.